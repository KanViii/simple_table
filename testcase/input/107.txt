INSERT x number
ASSIGN y 5