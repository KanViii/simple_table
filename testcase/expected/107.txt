Undeclared: ASSIGN y 5